docker/gcp_pudl_etl.sh

```diff
@@ -282,6 +289,9 @@ if [[ $ETL_SUCCESS == 0 ]]; then
         # Distribute Parquet outputs to a private bucket
         distribute_parquet 2>&1 | tee -a "$LOGFILE"
         DISTRIBUTE_PARQUET_SUCCESS=${PIPESTATUS[0]}
+        # Create duckdb file from parquet files
+        create_duckdb 2>&1 | tee -a "$LOGFILE"
```

Comment:
`create_duckdb()` or `create_and_distribute_duckdb()`?

(I have been loving the [ShellCheck linter for VSCode](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck))